### PR TITLE
update cloudcanal load

### PR DESCRIPTION
**Pull request by mengtingluo**
update cloudcanal load, change link,text and picture.

### CODE REVIEW

**Comment by evelynzhaojie** [comment]
this is your first pr in the StarRocks project, thanks for your contribution

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport 3.1 3.0 2.5

File: loading/CloudCanal_loading.md [hunk 1]
```diff
@@ -62,7 +66,7 @@ CloudCanal 提供了完整的产品化能力，用户在可视化界面完成数
 ![image.png](../assets/3.11-3.png)
 
 - 选择 **增量同步**，并且启用 **全量数据初始化**
-- 不勾选 DDL 同步(暂不支持)
+- 勾选 DDL 同步
```
Comment (hellolilyliuyi) [inline loading/CloudCanal_loading.md:69]: 这个对sr版本有要求吗？

Comment (mengtingluo) [reply loading/CloudCanal_loading.md:69]: 没有要求，都支持的

**Pull request merged**